<launch>
  <node name="usb_cam_defisheye" pkg="usb_cam_defisheye" type="usb_cam_defisheye_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="color_format" value="yuv422p" />
    <param name="camera_frame_id" value="usb_cam_defisheye" />
    <param name="io_method" value="mmap"/>
  </node>
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam_defisheye/image_raw"/>
    <param name="autosize" value="true" />
  </node>
</launch>
